t/prometheus-metrics.t

```diff
@@ -231,7 +271,7 @@ In particular, it shows the status codes and the response times
 --- response_body_like eval
 [
 "",
-qr/upstream_response_time_seconds(.|\n)*upstream_response_time_seconds_bucket\{le=".*"\} 1(.|\n)*upstream_status\{status="200"\} 1/
+qr/upstream_response_time_seconds(.|\n)*upstream_response_time_seconds_bucket\{service="",le=".*"\} 1(.|\n)*upstream_status\{status="200",service=""\} 1/
```

Comment:
when expected_response_body_like_multiple is available it might be worth to refactor this?